+srv/expServer.m

```diff
@@ -39,6 +39,7 @@ function expServer(useTimelineOverride, bgColour)
 rewardToggleKey = KbName('w');
 rewardPulseKey = KbName('space');
 rewardCalibrationKey = KbName('m');
+viewCalibrationKey = KbName('k');
```

Comment:
What do you think about changing this to a 'v' and including the gamma calibrations in the display?

Reply:
That seems reasonable to me! I think the water calibration is the thing the techs look at the most, but making more information more accessible seems like a good thing.

Reply:
Great, I'll change it to 'v' and make an issue about adding other plots in the future.